en/intro-concepts.md

```diff
@@ -120,11 +120,12 @@ communication. A fabric is a logical grouping of unique VLANs. A default fabric
 
 ## Spaces
 
-A *space* is a logical grouping of subnets that are able to communicate
-with each other. Subnets within each space need not belong to the same fabric.
-A default space ('space-0') is created when MAAS is installed and includes all
-detected subnets.
+A *space* is a logical grouping of VLANs whose subnets are able to communicate
+with one another. VLANs within each space need not belong to the same fabric.
+A default space is not created when MAAS is installed.
 
+A space's main purpose is to facilitate machine acquisition for
+[Juju](https://jujucharms.com/docs/stable/about-juju).
```

Comment:
Just a small suggestion - do you think it's worth explaining (briefly) _how_ spaces facilitate Juju?